extensions/amp-access/0.1/amp-access.js

```diff
@@ -950,14 +965,12 @@ class AccessTypeAdapterDef {
 }
 
 
-/**
- * @param {!Window} win
- * @return {!AccessService}
- */
-export function installAccessService(win) {
-  return getService(win, 'access', () => {
-    return new AccessService(win).start_();
-  });
-};
-
-installAccessService(AMP.win);
+// Register the extension services.
+AMP.extension(TAG, '0.1', function(AMP) {
+  AMP.registerServiceForDoc(
```

Comment:
In this particular case, we don't use fat arrows. This is specific to `AMP.extension()` call because it gets completely stripped by our code rewriter.